src/containers/ProfilePage.jsx

```diff
@@ -89,6 +95,20 @@ class ProfilePage extends React.Component {
     });
   }
 
+  checkPermissions() {
+    const { invalidPermission } = this.state;
+    if (Config.b2b.enable && invalidPermission) {
+      return (
+        <div className="message-permission">
+          <h2>{intl.get('permission-message')}</h2>
+        </div>
+      );
+    }
+    return (
+      <div className="loader" />
+    );
+  }
+
   render() {
     const { profileData } = this.state;
     const email = profileData && profileData._emails[0]._element ? profileData._emails[0]._element[0].email : '';
```

Comment:
```suggestion
    const email = profileData && profileData._emails && profileData._emails[0]._element ? profileData._emails[0]._element[0].email : '';
```

Reply:
There are multiple checks that need to be made here. `profileData ` will get returned but it will be missing sub-resources. Those need to be checked. Right now this will result in a broken page:
![image](https://user-images.githubusercontent.com/39800563/53824397-b3d78b00-3f28-11e9-8811-9fae09cac500.png)